cmd/health-monitor.go

```diff
@@ -0,0 +1,178 @@
+   |     |  |      |     | |     | |     |     | |       |            |
+   |     |  |      |     | |     | |     |     | |       |            |
+   | --- |  |      |     | |-----| |---- |     | |-----| |-----  ------
+   |     |  |      |     | |     | |     |     |       | |       |
+   | ____|  |_____ | ____| | ____| |     |_____|  _____| |_____  |_____
+
+
+   Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+
+   Copyright © 2020-2022 Microsoft Corporation. All rights reserved.
+   Author : <blobfusedev@microsoft.com>
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to deal
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in all
+   copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+   SOFTWARE
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/Azure/azure-storage-fuse/v2/common/config"
+	"github.com/Azure/azure-storage-fuse/v2/common/log"
+	"github.com/Azure/azure-storage-fuse/v2/component/file_cache"
+	hmcommon "github.com/Azure/azure-storage-fuse/v2/tools/health-monitor/common"
+	"github.com/spf13/cobra"
+)
+
+type monitorOptions struct {
+	EnableMon       bool     `config:"enable-monitoring"`
+	DisableList     []string `config:"monitor-disable-list"`
+	BfsPollInterval int      `config:"stats-poll-interval-sec"`
+	ProcMonInterval int      `config:"process-monitor-interval-sec"`
+}
+
+var pid string
+var cacheMonitorOptions file_cache.FileCacheOptions
+
+func resetMonitorOptions() {
+	options.MonitorOpt = monitorOptions{}
+	cacheMonitorOptions = file_cache.FileCacheOptions{}
+}
+
+var healthMonCmd = &cobra.Command{
+	Use:               "health-monitor",
+	Short:             "Monitor blobfuse2 mount",
+	Long:              "Monitor blobfuse2 mount",
+	SuggestFor:        []string{"healthmon", "monitor health"},
+	Args:              cobra.ExactArgs(0),
+	Hidden:            true,
+	FlagErrorHandling: cobra.ExitOnError,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		resetMonitorOptions()
+
+		err := validateMonOptions()
+		if err != nil {
+			log.Err("health-monitor: [%v]", err)
```

Comment:
Where are we initializing logging lib here ?

Reply:
This is part of the blobfuse cmd package. So, we don't need to initialize logging library separately here.